docs/software-architecture-and-design.md

```diff
@@ -396,9 +386,9 @@ Because of our [Coordinate Conventions](#coordinates), this means that an angle
 
 
 # Architecture Overview
-At a high-level our system is made of 4 main components: The [Backend](#backend), the [Sensor Fusion](#sensor-fusion), the [AI](#ai), and the [Visualizer](#visualizer). These 4 components each run in their own thread, and communicate with each other using the [Observer design pattern](#observer-design-pattern).
+At a high-level our system is made of 4 main components: The [Backend](#backend), the [Sensor Fusion](#sensor-fusion), the [AI](#ai), and [Thunderscope](#thunderscope). These 4 components each run in their own thread, and communicate with each other using the [Observer design pattern](#observer-design-pattern).
```

Comment:
Technically, Thunderscope doesn't use the Observer pattern in the way we mean here. Thunderscope uses a message queue architecture whenever data is published by the AI